Rocket.Chat/Controllers/Language/LanguageViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  LanguageViewController.swift
+//  Rocket.Chat
+//
+//  Created by Artur Rymarz on 26.02.2018.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+final class LanguageViewController: UIViewController {
+    private let viewModel = LanguageViewModel()
+    @IBOutlet weak var resetButton: UIButton! {
+        didSet {
+            resetButton.setTitle(viewModel.resetLabel, for: .normal)
+            resetButton.addTarget(self, action: #selector(resetLanguage), for: .touchUpInside)
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = viewModel.title
+    }
+
+    @objc private func resetLanguage() {
+        UserDefaults.standard.removeObject(forKey: "AppleLanguages")
```

Comment:
I believe this key name could be an static string. What do you think? I can see that's being used in lots of places.

Reply:
Yeah, indeed, will fix that.